server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -129,6 +132,12 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
     // when set, messages will be nacked based on the provided info
     private nackMessages: INackMessagesControlMessageContents | undefined;
 
+    // Session level properties
+    private scribeStartSuccess: boolean = false;
+    private serviceSummaryGenerated: boolean = false;
+    private sessionMetric: Lumber<LumberEventName.SessionResult> | undefined;
+    private sessionStartMetric: Lumber<LumberEventName.SessionResult> | undefined;
```

Comment:
Do we want to reuse the same `LumberEventName` (`SessionResult`) for both metrics? From below, it seems we have different use cases for `sessionMetric` vs `sessionStartMetric`. Do we also want to separate them in the telemetry?